### PR TITLE
[swift] Expose a few helper methods from SwiftGenerator

### PR DESCRIPTION
**Pull request by dnkoutso**
These can be useful for custom schema handlers related to Swift.

### CODE REVIEW

File: wire-swift-generator/api/wire-swift-generator.api
Comment (dnkoutso) [inline wire-swift-generator/api/wire-swift-generator.api:17]: do these look right? not sure

File: wire-tests-swift/no-manifest/src/main/swift/SwiftEdgeCases.swift
Comment (dnkoutso) [inline wire-tests-swift/no-manifest/src/main/swift/SwiftEdgeCases.swift:9]: @lickel the fact that I consolidated this under a single helper method it now affected enums just like it did messages, does it look good to you as a change?

Comment (dnkoutso) [reply wire-tests-swift/no-manifest/src/main/swift/SwiftEdgeCases.swift:9]: it also looks like enums called `Type` are also emitted `Type_` so this looks more aligned now

Comment (lickel) [reply wire-tests-swift/no-manifest/src/main/swift/SwiftEdgeCases.swift:9]: Nice

**Review by lickel — APPROVED** [review]

File: wire-swift-generator/api/wire-swift-generator.api
Comment (dnkoutso) [reply wire-swift-generator/api/wire-swift-generator.api:15]: I spoke to @oldergod offline about this to lead up to this PR but yes will wait for the approval here.

Comment (dnkoutso) [reply wire-swift-generator/api/wire-swift-generator.api:15]: @oldergod can we specify the name to expose for this tool so its not so ugly `getBUILT_IN_TYPES`?

Comment (oldergod) [reply wire-swift-generator/api/wire-swift-generator.api:15]: You can add `@JvmStatic` to the variable.

But instead of exposing the list of built-in types, could we just expose a method like `fun builtInType(protoType: ProtoType): Boolean = protoType in BUILT_IN_TYPES.keys` ?

Comment (dnkoutso) [reply wire-swift-generator/api/wire-swift-generator.api:15]: ha that is already exposed, ok will update this `fun builtInType(protoType: ProtoType): Boolean = protoType in BUILT_IN_TYPES.keys`

**Review by oldergod — APPROVED** [review]

**Pull request merged**
